api/python/provisioner/api.py

```diff
@@ -296,6 +296,33 @@ def sw_update(targets=ALL_MINIONS, nowait=False):
     )
 
 
+def rollback_update(target_version, targets=ALL_MINIONS, nowait=False):
+    r"""Rollback CORTX software stack.
+
+    Rollback all CORTX components to the specified target_version.
+
+    Assumptions - rollback pillar is updated, with specified target_version
+    as 'cortx_version' and yum txn ids for each node
+    rollback:
+      <cortx-version>:
+        <node-id-x>:
+          yum_txn_id: <ID>
+
+    Parameters
+    ----------
+    target_version
+        Rollback target old version
+    targets
+        (optional) A host to rollback. Default: all minions
+    nowait
+        (optional) Run asynchronously. Default: False
+    """
+
+    return _api_call(
+        'rollback_update', target_version, targets=targets, nowait=nowait
```

Comment:
Should we change this to
**sw_rollback**

Reply:
addressed